core/arch/riscv/kernel/sbi.c

```diff
@@ -40,6 +57,20 @@ void sbi_console_putchar(int ch)
 	sbi_ecall(SBI_EXT_0_1_CONSOLE_PUTCHAR, 0, ch);
 }
 
+/**
+ * sbi_dbcn_write_byte() - write byte to debug console
+ * @ch:         byte to be written
```

Comment:
s/byte/Byte/ for consistency with the other function inline description comments.
Remove also the inner indentation, also for consistency in this source file.